tests/foreman/maintain/test_backup_restore.py

```diff
@@ -46,22 +46,11 @@
 assert_msg = "Some required backup files are missing"
 
 
-def get_exp_files(sat_maintain, backup_type):
-    if type(sat_maintain) is Satellite:
-        if sat_maintain.is_remote_db():
-            expected_files = BASIC_FILES | ONLINE_SAT_FILES
-        else:
-            expected_files = (
-                BASIC_FILES | OFFLINE_FILES
-                if backup_type == 'offline'
-                else BASIC_FILES | ONLINE_SAT_FILES
-            )
-    else:
-        expected_files = (
-            BASIC_FILES | OFFLINE_FILES
-            if backup_type == 'offline'
-            else BASIC_FILES | ONLINE_CAPS_FILES
-        )
+def get_exp_files(sat_maintain, backup_type, skip_pulp=False):
+    is_remote_db = type(sat_maintain) is Satellite and sat_maintain.is_remote_db()
+    expected_files = ONLINE_SAT_FILES if backup_type == 'online' or is_remote_db else OFFLINE_FILES
```

Comment:
Seems that one more condition for capsule files is needed.
Also it may be worth to add a comment that only online backup is available for remote db mode.

Reply:
Good catch.
I've just noticed it too.

Reply:
Updated